src/components/v5/shared/Navigation/ColonySwitcher/partials/TitleSections/BaseTitleSection.tsx

```diff
@@ -0,0 +1,28 @@
+import { type IconProps } from '@phosphor-icons/react';
+import React, { type ComponentType, type FC } from 'react';
+
+interface BaseTitleSectionProps {
+  title: string;
+  description: string;
+  icon: ComponentType<IconProps>;
+}
+
+export const BaseTitleSection: FC<BaseTitleSectionProps> = ({
```

Comment:
I like this component! Big fan of making a generic one like this and then making specific components which just call the generic one with props 🤙